ginivision/src/main/res/layout-sw600dp-land/gv_activity_photo_tips.xml

```diff
@@ -0,0 +1,194 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@color/gv_photo_tips_activity_background"
+    android:orientation="vertical">
+
+    <net.gini.android.vision.internal.ui.CustomFontTextView
+        android:id="@+id/gv_photo_tips_headline"
+        style="@style/GiniVisionTheme.Help.PhotoTips.Header.TextStyle"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_alignParentTop="true"
+        android:layout_alignLeft="@+id/gv_photo_tips_scrollview"
+        android:layout_alignStart="@+id/gv_photo_tips_scrollview"
+        android:padding="16dp"
+        android:text="@string/gv_photo_tips_headline" />
+
+    <ScrollView
+        android:id="@+id/gv_photo_tips_scrollview"
+        android:layout_width="wrap_content"
+        android:layout_height="match_parent"
+        android:layout_alignParentBottom="true"
+        android:layout_below="@+id/gv_photo_tips_headline"
+        android:layout_centerHorizontal="true"
+        android:overScrollMode="ifContentScrolls"
+        android:scrollbars="none">
+
+        <GridLayout
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_gravity="center_horizontal"
+            android:columnCount="2"
+            android:orientation="horizontal">
+
+            <LinearLayout
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_gravity="center"
+                android:layout_marginEnd="32dp"
```

Comment:
Since this value is repeated several times, should it be moved to a _dimen_ constant?